src/platform/forms/components/review/SubmitButtons.jsx

```diff
@@ -0,0 +1,45 @@
+// libs
+import React from 'react';
+
+// platform - forms-system components
+import Column from 'platform/forms/components/common/grid/Column';
+import ProgressButton from 'platform/forms-system/src/js/components/ProgressButton';
+import Row from 'platform/forms/components/common/grid/Row';
+
+// platform - forms containers
+import PreSubmitSection from 'platform/forms/containers/review/PreSubmitSection';
+
+function SubmitButtons(props) {
+  const { goBack, formConfig, onSubmit } = props;
+
+  return (
+    <>
+      <Row>
+        <Column />
+      </Row>

```

Comment:
Why is this empty? In the [old version](https://github.com/department-of-veterans-affairs/vets-website/blob/master/src/platform/forms-system/src/js/review/SubmitButtons.jsx#L158-L160), this rendered the submit message.
```js
      <div className="row">
        <div className="columns">{submitMessage}</div>
      </div>
```

Reply:
Correct, but for the case in which we are just rendering the buttons `(submission.status === false)` and not setting the `submitMessage`. So in respect to keeping the same html footprint as prior to the refactor, we have that there. 

`let submitMessage;`